homeassistant/components/automation/websocket_api.py

```diff
@@ -61,10 +61,21 @@ def websocket_automation_trace_get(hass, connection, msg):
 
 @callback
 @websocket_api.require_admin
-@websocket_api.websocket_command({vol.Required("type"): "automation/trace/list"})
+@websocket_api.websocket_command(
+    {vol.Required("type"): "automation/trace/list", vol.Optional("automation_id"): str}
+)
 def websocket_automation_trace_list(hass, connection, msg):
     """Summarize automation traces."""
-    automation_traces = get_debug_traces(hass, summary=True)
+    automation_id = msg.get("automation_id")
+
+    if not automation_id:
```

Comment:
Hmm need to think if we want it like this. Until now we didn't have an endpoint that would return a different shape of data based on the parameters.  But having a dict `{automation_id: traces}` also feels a bit weird.

Reply:
Should we make it two endpoints instead?

Reply:
oh wait now it doesn't matter anymore, both are a list now.